app/templates/frameworks/_guidance_links.html

```diff
@@ -1,87 +1,104 @@
-        </a>
-        {% if last_modified.supplier_updates %}
-        <div class="hint">
+    {% if communications_files.proposed_agreement.last_modified %}
+    <p>
+      <a href="{{ url_for('.download_supplier_file', filepath=communications_files.proposed_agreement.filename, framework_slug=framework.slug) }}">
+        Download the proposed framework agreement
+      </a>
+      <br/>
+      Read the proposed agreement between the Crown Commercial Service (CCS) and suppliers.
+      <br/>
+      <span class="hint">
         Last updated
-        <time datetime="{{ last_modified.supplier_updates }}">
-            {{ last_modified.supplier_updates|dateformat }}
+        <time datetime="{{ communications_files.proposed_agreement.last_modified }}">
+          {{ communications_files.proposed_agreement.last_modified|dateformat }}
         </time>
-        </div>
-        {% endif %}
-    </li>
-  </ul>
-{% else %}
-  <h2>
-    Guidance, updates and questions
-  </h2>
-  <ul class="list-no-bullet">
-    <li>
-      <a href="{{ url_for('.download_supplier_file', filepath=supplier_pack_filename, framework_slug=framework.slug) }}">
-        <span>Download guidance and legal documentation (.zip)</span>
+      </span>
+    </p>
+    {% endif %}
+
+    {% if communications_files.call_off.last_modified %}
+    <p>
+      <a href="{{ url_for('.download_supplier_file', filepath=communications_files.call_off.filename, framework_slug=framework.slug) }}">
+        Download the proposed ‘call-off’ contract
       </a>
-      {% if last_modified.supplier_pack %}
-      <div class="hint">
+      <br/>
+      Read the proposed contract between a buyer and a supplier.
+      <br/>
+      <span class="hint">
         Last updated
-        <time datetime="{{ last_modified.supplier_pack }}">
-          {{ last_modified.supplier_pack|dateformat }}
+        <time datetime="{{ communications_files.call_off.last_modified }}">
+          {{ communications_files.call_off.last_modified|dateformat }}
         </time>
-      </div>
-      {% endif %}
-    </li>
+      </span>
+    </p>
+    {% endif %}
+  </div>
+{% endif %}
+  <div>
+    <h2 class="heading-xmedium" class="heading-xmedium">Communications</h2>
 
-    <li>
+    {% if framework.status not in ['standstill', 'live'] %}
+    <p>
+      You can review and ask clarification questions about this
+      framework until {{ framework_dates.clarifications_close_date }}.
+    </p>
+    {% endif %}
+
+    {% block updates_para %}
+    <p>
       <a href="{{ url_for('.framework_updates', framework_slug=framework.slug) }}">
-        <span>
-          {% if framework.clarificationQuestionsOpen %}
-            Read updates and ask clarification questions
-          {% else %}
-            Read updates and clarification question responses
-          {% endif %}
-        </span>
+        View communications and
+        {% if framework.status not in ['standstill', 'live'] %}ask{% endif %}
```

Comment:
Again, you can only ask questions while it's `open`.  This would show "ask" during `pending` too, which we don't want.